src/frontend/src/common/analytics/eventTypes.ts

```diff
@@ -4,11 +4,168 @@
  * This exists to ensure consistency/clarity across types and to provide
  * a reference for data analysts working on CZ GE.
  *
- * - Each event type should have a plain-English description of what it means.
+ * --- REQUIREMENTS for adding new event types ---
+ * - Each event type should have a comment with a plain-English description
+ *   of what the event means.
  * - Event types should be written in SCREAMING_SNAKE_CASE
  * - Each entry in the enum should be a duplicate key/value string enum
+ * - If an event type entails sending additional data about what happened
+ *   (most events do), there should be a matching event TypeScript type below
+ *   in this file that shows the structure of the additional event data.
+ *   => The TS type should be named according to the event type as follows:
+ *      Event of `SOME_EVENT_FOO` <--> TS type of `AnalyticsSomeEventFoo`
  */
 export enum EVENT_TYPES {
   // User has been sent to Nextstrain to view a phylo tree
   TREE_VIEW_NEXTSTRAIN = "TREE_VIEW_NEXTSTRAIN",
+
+  // User has kicked off the creation of a Nextstrain phylo tree
+  TREE_CREATION_NEXTSTRAIN = "TREE_CREATION_NEXTSTRAIN",
+
+  // User has been sent over to UShER site. UShER will now create tree and
+  // display it on their site. External, so can't tell if tree succeeds/fails.
+  // **No `additionalEventData`**. Just tracking event occurrence right now.
+  TREE_CREATION_VIEW_USHER = "TREE_CREATION_VIEW_USHER",
+
+  // User has downloaded a file of the actual phylo tree
+  TREE_DOWNLOAD_TREE_FILE = "TREE_DOWNLOAD_TREE_FILE",
```

Comment:
nit: I would say a better name for this would be `TREE_DOWNLOAD_JSON_FILE`

Reply:
So here's my thinking. I considered naming it that when I was working, but I wound up thinking it was better to avoid naming the file type in case of later changes/addition. I know that right now all trees come as JSON, but it's conceivable another file type (YAML or maybe something totally new) could come on the scene  eventually. It felt like something where you can name the implementation or name the concept. If we start offering different filetypes down the road, we could then add a `tree_file_type` key to the event that names which one, but right now it's just JSON, so we don't need that yet.

That said, it sure is awkward naming! We're downloading from the category of tree stuff, and we're downloading a file of the tree. I think the best name would be `TREE_DOWNLOAD_FILE`, but it's confusing, because we also have other file downloads around trees (eg, `TREE_DOWNLOAD_SELECTED_SAMPLES_TEMPLATE`), so the nice name is too generic, because it gets confused with the other flavors of download involved in tree stuff.

I'm basically looking at it like this `TREE_DOWNLOAD` means we're downloading something from the category of tree stuff, we're on the tree table, etc. Then the next part if what we're downloading from there, and we are downloading the file representation of the tree, so `TREE_FILE`.